tachyon/zk/lookup/halo2/BUILD.bazel

```diff
@@ -18,20 +18,10 @@ tachyon_cc_library(
 )
 
 tachyon_cc_library(
-    name = "permute_expression_pair",
-    hdrs = ["permute_expression_pair.h"],
-    deps = [
-        "//tachyon/zk/base/entities:prover_base",
-        "//tachyon/zk/lookup:lookup_pair",
-        "@com_google_absl//absl/container:btree",
-    ],
-)
-
-tachyon_cc_library(
-    name = "prover",
+    name = "lookup_prover",
```

Comment:
Given that the `Prover` is enclosed within the `lookup` namespace, I initially intended to name it simply `Prover. Would `lookup::LookupProver` not be redundant in this context?

Moreover, having the prefix `lookup_` doesn't seem natural, especially considering that only the `prover.h` file has been changed to `lookup_prover.h`, while the other files remain under the lookup directory.